File: stage-02th/src/main/java/cn/uncleyumo/chapter03/unit01/Test03.java
package cn.uncleyumo.chapter03.unit01;

import java.util.Arrays;

/**
 * @author dev.uncleyumo.cn
 * @fileName Test03
 * @createDate 4/22/25 April
 * @school Wuxi University
 * @studentID 22344131
 * @description
 */

public class Test03 {
    public static void main(String[] args) {
        Integer[] arr = {2, 4, 6, 1, 9, 3, 0, 7};
//        Arrays.sort(arr, new Comparator<Integer>() {
//            @Override
//            public int compare(Integer i1, Integer i2) {
//                return i2.compareTo(i1);
//            }
//        });
        Arrays.sort(arr, (i1, i2) -> i2.compareTo(i1));
        System.out.println(Arrays.toString(arr));
    }
}

File: stage-01th/src/main/java/cn/uncleyumo/chapter03/kotlin/Test03.kt
package cn.uncleyumo.chapter03.kotlin

/**
 * @author uncle_yumo
 * @fileName Test03
 * @createDate 2025/4/16 April
 * @school 无锡学院
 * @studentID 22344131
 * @description
 */

fun main() {
    val food01 = Food()
    val food02 = Food()
    println(food01 === food02)
    println(food01 == food02)
    println(System.identityHashCode(food01))
    println(System.identityHashCode(food02))
}